criu/sockets.c

```diff
@@ -605,14 +605,18 @@ int restore_socket_opts(int sk, SkOptsEntry *soe)
 	tv.tv_usec = soe->so_rcv_tmo_usec;
 	ret |= restore_opt(sk, SOL_SOCKET, SO_RCVTIMEO, &tv);
 
-	ret |= restore_bound_dev(sk, soe);
```

Comment:
restore_socket_opts was called for packet sockets too. Do we need to call restore_bound_dev() for them with your change?

Reply:
restore_bound_dev() method should be retained here, it's also need by packet socket.